test/jdk/java/net/httpclient/UserCookieTest.java

```diff
@@ -0,0 +1,573 @@
+/*
+ * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should the year be 2021?

Reply:
Darn. I thought I'd already fixed that. Good catch!